src/main/java/com/dataloom/directory/pojo/Auth0UserBasic.java

```diff
@@ -97,7 +103,7 @@ public String getUsername() {
 
     @Override
     public int hashCode() {
-        //user id is a unique identifier.
+        // user id is a unique identifier.
```

Comment:
If userId is enforced to be not null (it should be) then you can just return userId.hashCode